doc/emacs-guix.texi

```diff
@@ -114,6 +114,10 @@ Shell completions for all guix commands and options
 @item
 Minor mode with additional functionality for @code{scheme-mode} to
 work with Guix Scheme files (@pxref{Development}).
+
+@item
+Major mode for @code{sh-mode} to work with Guix environment-variables
+and profile files (@pxref{Development}).
```

Comment:
I think ``(@pxref{Development})`` should not be here.  Also "major mode for sh-mode" sounds strange to me (as `sh-mode` is also a major mode).  Maybe "based on sh-mode" is better or just remove this bit about sh-mode, if you are going to add a chapter documenting guix-env-var-mode in more details.